morphuntion/build.gradle

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020-2024 Apple Inc. All rights reserved.
+ */
+
+plugins {
+    id 'net.ltgt.errorprone' version '1.1.1' apply false
+    id 'maven-publish'
```

Comment:
It's dictionary-parser.  Denny will likely want to look at that code. It can be reimplemented in another programming language when using wikidata as a data source, but I wanted to provide the tool source code so that the logic behind generating the lexical dictionaries is a lot clearer.  The tool will likely be deleted once wikidata can be ingested.

This also had a Java wrapper through JNI. That implementation was excluded from this contribution at this time to expedite making this implementation open source.